.travis.yml

```diff
@@ -0,0 +1,13 @@
+language: php
+php:
+  - '5.5'
+  - '5.6'
+sudo: required
+dist: trusty
+#before_script: phpenv config-add php-custom-config.ini
```

Comment:
Left over from when I originally was trying to figure out how to make travis work, will remove